woof-code/rootfs-packages/ptheme/usr/sbin/ptheme

```diff
@@ -46,7 +46,7 @@ build_menus (){
 	MENUITEMS_JWM_COLOR="$(find /usr/share/jwm/themes/ -mindepth 1 -maxdepth 1 -iname "*jwmrc" -printf "<menuitem label=\"%f\"><action>echo %f > $WORKDIR/ptheme-jwm</action><action>refresh:PTHEME_JWM_COLOR</action></menuitem>" | sort | sed -e 's/-jwmrc//g' &)"
 	MENUITEMS_JWM_TRAY="$(build_menu /usr/share/jwm/tray_templates $WORKDIR/ptheme-jwm_tray PTHEME_JWM_TRAY)"
 	MENUITEMS_JWM_BUTTONS="$(build_menu /usr/share/jwm/themes_window_buttons/ $WORKDIR/ptheme-jwm_buttons PTHEME_JWM_BUTTONS)"
-	MENUITEMS_MOUSE="$(find -L $HOME/.icons -mindepth 1 -maxdepth 1 -type d -printf "<menuitem label=\"%f\"><action>echo %f > $WORKDIR/ptheme-mouse</action><action>refresh:PTHEME_MOUSE</action></menuitem>" | sort | grep -vE "ROX|default" &)"
+	MENUITEMS_MOUSE="$(find $HOME/.icons -mindepth 1 -maxdepth 1 -type d -printf "<menuitem label=\"%f\"><action>echo %f > $WORKDIR/ptheme-mouse</action><action>refresh:PTHEME_MOUSE</action></menuitem>" | sort | grep -vE "ROX|default" &)"
```

Comment:
Why is this -L removed?

Reply:
https://github.com/puppylinux-woof-CE/woof-CE/commit/0e9de2dcbac3daf4aa53164aa782c0a72f7d1d2f